src/main/java/duke/Output.java

```diff
@@ -0,0 +1,135 @@
+package duke;
```

Comment:
I like how you have a class just for the output :)

Reply:
Thanks!